nixos/modules/services/web-apps/goatcounter.nix

```
@@ -0,0 +1,57 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib) types;
+  cfg = config.services.goatcounter;
+  stateDir = "goatcounter";
+in
+
+{
+  options = {
+    services.goatcounter = {
+      enable = lib.mkEnableOption "Easy web analytics. No tracking of personal data.";
```

```suggestion
      enable = lib.mkEnableOption "goatcounter";
```
Otherwise you get _"Whether to enable Easy web analytics. No tracking of personal data…":

---

done